src/DssSpell.sol

```diff
@@ -42,8 +66,47 @@ contract DssSpellAction is DssAction {
     function actions() public override {
         // ----- Deploy Multiswap Conduit for RWA015-A -----
 
+        // ----- Add previous CRON KEYS -----
+
+        // NOT FOR GOERLI
+        // DssExecLib.setChangelogAddress("CRON_SEQUENCER", CRON_SEQUENCER);
+        // DssExecLib.setChangelogAddress("CRON_AUTOLINE_JOB", CRON_AUTOLINE_JOB);
+        // DssExecLib.setChangelogAddress("CRON_LERP_JOB", CRON_LERP_JOB);
+        // DssExecLib.setChangelogAddress("CRON_D3M_JOB", CRON_D3M_JOB);
+        // DssExecLib.setChangelogAddress("CRON_CLIPPER_MOM_JOB", CRON_CLIPPER_MOM_JOB);
+        // DssExecLib.setChangelogAddress("CRON_ORACLE_JOB", CRON_ORACLE_JOB);
+
         // ----- Deploy FlapperUniV2 -----
 
+        DssExecLib.setChangelogAddress("PIP_MKR", PIP_MKR);
+
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
```

Comment:
Can we add similar comments like these (link to poll + commit of init scripts):
https://github.com/makerdao/spells-goerli/blob/2023-07-12-flapper/archive/2023-04-28-DssSpell/DssSpell.sol#L77

Reply:
Done!